src/module.js

```diff
@@ -370,6 +374,14 @@ export default class DailyIframe extends EventEmitter {
         };
       }
     }
+    // default the local video + participants bar display to true if not specified
```

Comment:
shoot, just realized that by setting these default properties here, we're missing the `wrap()` and `createTransparentFrame()` factory methods. i'm guessing we can ignore the latter since it's for custom CSS, but I think `wrap()` is still relevant.

regardless, hopefully it's not too hard to move this defaulting logic to the constructor (like on line 473, below).

Reply:
totally right, thanks for noticing that!